### PR TITLE
Added recursive functionality to list view

### PR DESCRIPTION
**Pull request by abhishekpratapa**
- Added recursive functionality to lists.

### CODE REVIEW

**Comment by abhishekpratapa** [comment]
fixed #440

File: src/unity/lib/unity_sframe.cpp [hunk 1]
```diff
@@ -1840,8 +1934,11 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
                 }
                 break;
               case flex_type_enum::LIST:
-                ss << value.to<std::string>();
-                break;
+                {
```
Comment (znation) [inline src/unity/lib/unity_sframe.cpp]: Shouldn't this whole section (switch/case over type, with stringification) get refactored into `listHelper`? Seems like almost identical logic.

File: src/unity/lib/unity_sframe.cpp [hunk 2]
```diff
@@ -1673,7 +1673,101 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       ew << ss.str();
     }
 
-    auto getRows = [self, &reader, &ew, &column_names, &empty_tz, &image_queue](size_t start, size_t end) {
+    std::function<std::string(flexible_type&)> listHelper;
+    listHelper = [&listHelper, &empty_tz](flexible_type& value) {
```
Comment (znation) [inline src/unity/lib/unity_sframe.cpp]: Can we come up with a better name than `listHelper`? How about `escapeForTable`? (Is this only for the accordion, only for inline display in the table, or both? Let's pick a name that communicates the answer to that question...)

Comment (abhishekpratapa) [reply src/unity/lib/unity_sframe.cpp]: Just for the inline display in the table. How's `escapeForInlineTable`?

Comment (znation) [reply src/unity/lib/unity_sframe.cpp]: I find the word `inline` confusing. I think `escapeForTable` is fine.

**Pull request merged**